server/api/lib/mavat/index.js

```diff
@@ -182,27 +183,19 @@ const fetch = (planUrl, fetchPlanInstruction = true) =>
 const fetchPlanData = (planUrl) =>
 	new Promise((resolve, reject) => {
 		(async () => {
-			const page = await browser.newPage();
+			
 
 			try {
 				Log.debug('Loading plan page', planUrl);
 				try {
-					await page.goto(planUrl);
-					await page.waitForSelector('body > pre');
-					const jsonContent = await page.evaluate(
-						() => document.getElementsByTagName('pre')[0].innerText
-					);
-
+					const jsonContent = await proxy.get(planUrl);
```

Comment:
Hi, did it work? Also, we though the scrapping be could be a fallback solution, meaning we should do that on the catch, didn't we?

Reply:
Yes, it works really well!

Uh oh. You are right. I totally forgot that we said it would be a fallback. Let me fix that and also the tests.. will lyk when ready to review again.

Reply:
Ok, I changed it to use scraping bee as fallback. This also fixed the server tests. Please review again 🙏🏽

Reply:
great!!! I really wish we had this one before as large plans already failed.